test/jdk/tools/jmod/hashes/HashesTest.java

```diff
@@ -401,8 +386,7 @@ private ModuleHashes hashes(String name) {
     private ModuleHashes hashes(ModuleFinder finder, String name) {
         ModuleReference mref = finder.find(name).orElseThrow(RuntimeException::new);
         try {
-            ModuleReader reader = mref.open();
-            try (InputStream in = reader.open("module-info.class").get()) {
+            try (ModuleReader reader = mref.open(); InputStream in = reader.open("module-info.class").get()) {
```

Comment:
Thats for doing the right thing and using BeforeMethod. The changes looks good.
There is some random re-formatting in hashes and deleteDirectory, I don't know if this was intended or not. I think we should at least resolve L389 before integrating.

Reply:
Thank you Alan.  The re-formatting was done automagically by Intellij.  I just pushed an update to address your suggestions.